CRM/Speakcivi/Page/Speakcivi.php

```diff
@@ -180,6 +183,19 @@ public function share($param) {
 
   }
 
+  /**
+   * Create a representative mail activity
+   *
+   * @param $param
+   */
+  public function mail($param) {
+
+    $contact = $this->createContact($param);
+    $activity = $this->createActivity($param, $contact['id'], 'Email', 'Completed');
```

Comment:
What email? opt-in? we don't use optin's from speakcivi in here.

Targets are not contacts, it would perhaps be nice to add them and have a with: to them, but I am so incompetent as a PHP programmer that I have used all my mana for just this draft Rmail support. However, maybe some other Speakcivi developers will gen inspired by this first step!?